mergify_engine/engine/actions_runner.py

```diff
@@ -164,7 +197,9 @@ async def gen_summary(
             summary += STRICT_MODE_DEPRECATION_SASS
         else:
             summary += STRICT_MODE_DEPRECATION_GHES
-
+    summaryValue = _has_merge_action_with_commit_message_mode(pull_request_rules)
```

Comment:
We use snake cases everywhere
```suggestion
    summary_value = _has_merge_action_with_commit_message_mode(pull_request_rules)
```